scripts/components/Shared/HotspotNavButton.js

```diff
@@ -105,8 +105,13 @@ export default class HotspotNavButton extends React.Component {
     this.props.setHotspotValues(newSizeWidth, newSizeHeight)
   }
 
-  render() {
+  determineTabIndex = () => {
+    return this.context.extras.isEditor || this.props.isHotspotTabbable ? this.props.tabIndexValue : -1
```

Comment:
If we return `null` instead of a value, the attribute won't be rendered at all. Do we need `tabindex="-1"` anywhere?

```suggestion
    return this.context.extras.isEditor || this.props.isHotspotTabbable ? this.props.tabIndexValue : null;
```

Reply:
Yes, if the tabindex is not rendered, or not set to -1 in the viewer-context it's still tabbable.

Reply:
Ahh, because of the implicit tabindex, I get it. Disregard 🌼